src/api/app/views/webui/attribute/_issue_fields.html.haml

```diff
@@ -0,0 +1,6 @@
+.nested-fields
+  .field
+    = f.select :issue_tracker_id, IssueTracker.all.collect { |tracker| [ tracker.name, tracker.id ] }.sort, include_blank: false
```

Comment:
This PR is related to the "Convert ERB templates to HAML" fruit tree card if I have to refactor ALL ERB templates of OBS Webui by the way.... I won't do any other thing then for some months 😄 

The point of this is at least have all templates in HAML, not refactor everything I see.